TELCODOCS-2045 - Telco core RDS 4.18 docs (#88622)
openshift/openshift-docs: modules/telco-core-rds-product-version-use-model-overview.adoc

// Module included in the following assemblies:
//
// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc

:_mod-docs-content-type: REFERENCE
[id="telco-core-rds-product-version-use-model-overview_{context}"]
= Telco core RDS {product-version} use model overview

> The Telco core reference design specification (RDS) describes a platform that supports large-scale telco applications including control plane functions such as signaling and aggregation.

Reviewer: ```suggestion
The Telco core reference design specification (RDS) describes a platform that supports large-scale telco applications, including control plane functions such as signaling and aggregation.
```

> The Telco core reference design specification (RDS) describes a platform that supports large-scale telco applications, including control plane functions such as signaling and aggregation.

Reviewer 2: "Telco core" is sometimes capitalized and sometimes not.

> It also includes some centralized data plane functions, for example, user plane functions (UPF).

Reviewer: ```suggestion
It also includes some centralized data plane functions, such as user plane functions (UPF).
```
